openshift/system/Dockerfile

```diff
@@ -69,6 +70,7 @@ RUN export ${BUNDLER_ENV} >/dev/null \
     && umask 0002 \
     && cd /opt/system \
     && npm install -g yarn \
+    && yarn config set network-timeout 300000 \
```

Comment:
Why this timeout?

Reply:
It looks like it was an intermittent issue. Removed.